packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -541,6 +545,97 @@ void main() {
           ));
           handle.dispose();
         });
+
+        testWidgets('ReorderableListView still scrolls when physics property is set', (WidgetTester tester) async {
+          final List<Widget> widgetList = <Widget>[
+            Container(key: const Key('1'), child:  const Icon(Icons.account_circle))
+          ];
+          final SingleChildScrollView scrollView = SingleChildScrollView(
```

Comment:
Is there a reason to wrap the `ReorderableListView` in a `SingleChildScrollView`? The reason I'm asking is because `ReorderableListView` itself implements a `SingleChildScrollView`.